test/configuration-parser/test.js

```diff
@@ -3,7 +3,7 @@ var expect = require('chai').expect;
 var configParser = require('../../src/config-parser.js');
 var expectedResults = require('./test-results/results.js');
 
-require('mocha-sinon');
+require('mocha-sinon')();
```

Comment:
Fix for watch mode: https://github.com/elliotf/mocha-sinon#using-mochas-flakey---watch-flag

Reply:
>You can put the following block in your spec helper and it might work:
so much confidence :P

Reply:
:-D